nncf/common/sparsity/schedulers.py

```diff
@@ -277,7 +291,7 @@ def __init__(self, controller: SparsityController, params: dict):
         self.schedule = MultiStepSchedule(
             sorted(params.get('multistep_steps', [90])), params.get('multistep_sparsity_levels', [0.1, 0.5]))
         self.target_level = self.schedule.values[-1]
-        self._current_level = self.schedule.values[0]
+        self._current_level = None
```

Comment:
I think you can remove this line because the same is in the `SparsityScheduler` superclass.

Reply:
Thanks for the comment! Updated